capstone/src/main/webapp/image_analyzer.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="Christopher Mohri, Daniel Ortega and Oluwadamilare Alamutu">
+    <meta name="generator" content="Jekyll v3.8.5">
+    <title>Perspective Hub</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="assets/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="assets/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="assets/favicon-16x16.png" sizes="16x16">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="cover.css" rel="stylesheet">
+  </head>
+  <body class="text-center">
+    <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
+      <header class="masthead mb-auto">
+        <div class="inner">
+          <div class="text-center">
+            <img src="assets/perspective_horizontal_color.png" class="rounded">
+          </div>
+          <h3 class="masthead-brand">Perspective Hub</h3>
+          <nav class="nav nav-masthead justify-content-center">
+            <a class="nav-link" href="index.html">Home</a>
+            <a class="nav-link" href="text_analyzer.html">Text Analyzer</a>
+            <a class="nav-link active" href="image_analyzer.html">Image Analyzer</a>
+            <a class="nav-link" href="https://support.perspectiveapi.com/s/" target="_blank">Support</a>
+            <a class="nav-link" href="https://medium.com/the-false-positive" target="_blank">Blog</a>
+          </nav>
+        </div>
+      </header>
+      <h2>Attach an image and select the Perspective API attribute(s) you would like to be shown</h2>
+      <div class="input-group">
+        <div class="input-group-prepend">
+          <span class="input-group-text" id="inputFileText">Upload</span>
+        </div>
+        <div class="custom-file">
+          <input type="file" class="custom-file-input" id="imageForAnalysis" aria-describedby="inputFileText">
+          <label class="custom-file-label" for="imageForAnalysis">Select Image to be Analyzed</label>
+        </div>
+      </div>
+      <form>
+        <div class="form-check">
```

Comment:
Just a note, we should probably discuss the options for user input - right now, I think the back-end only supports receiving text and a language